tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -692,3 +692,68 @@ def forward_xp(self, inputs, xp):
 def test_swap_invalid(xp, shape, axis1, axis2):
     a = array_utils.create_dummy_ndarray(xp, shape, 'float32')
     return xp.swapaxes(a, axis1, axis2)
+
+
+@op_utils.op_test(['native:0'])
+@chainer.testing.parameterize_pytest('shape', [
+    (2, 2, 2),
+    (3, 3, 2, 3, 3),
+    (3, 0, 2, 0, 3),
+    (1, 2, 3, 1, 3, 3),
+    (3, 4, 5, 2, 3, 5),
+    (1,)
+])
+class TestExpandDIms(op_utils.NumpyOpTest):
```

Comment:
Could you parameterize `ndim` and `is_contiguous` too?

Reply:
Sorry, I made a mistake writing. `axis` is correct.

Reply:
> writing. `axis` is correct.

Shouldn't we check over all possible axis positions?

Reply:
If parameterizing `axis`, the error message tells us the axis of the failure cases when the test fails. We should check over all possible `axis`, of course.

Reply:
Have updated the tests.

Reply:
Thanks!